### PR TITLE
style: cleanup unused lll and '.' at the end of comments

### PR DESCRIPTION
**Pull request by AlisonB319**
**What type of PR is this?**

/kind cleanup
/kind failing-test

**What this PR does / why we need it**:
Cleaning up unused lll lint errors and periods at the end of comments errors. 

**Which issue(s) this PR fixes**:

(https://github.com/Azure/AgentBaker/actions/runs/6399258600)

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/vhdbuilder/datamodel/component_configs.go [hunk 1]
```diff
@@ -7,9 +7,9 @@ import (
 	"strings"
 )
 
-// Components is the data model for the component config
-// The component config is a JSON file
-// for example, vhdbuilder/packer/components.json
+// Components is the data model for the component config.
+// The component config is a JSON file.
```
Comment (cameronmeissner) [inline pkg/vhdbuilder/datamodel/component_configs.go:11]:
```suggestion
// Components is the data model for the component config.
// The component config is a JSON file, for example, vhdbuilder/packer/components.json.
```

Comment (cameronmeissner) [reply pkg/vhdbuilder/datamodel/component_configs.go:11]: same for the below

File: pkg/vhdbuilder/datamodel/component_configs.go [hunk 2]
```diff
@@ -7,9 +7,10 @@ import (
 	"strings"
 )
 
-// Components is the data model for the component config
-// The component config is a JSON file
-// for example, vhdbuilder/packer/components.json
+// Components is the data model for the component config.
```
Comment (cameronmeissner) [inline pkg/vhdbuilder/datamodel/component_configs.go:10]: dupe

File: pkg/vhdbuilder/datamodel/component_configs.go [hunk 3]
```diff
@@ -29,9 +30,9 @@ type DownloadFile struct {
 	TargetContainerRuntime string   `json:"targetContainerRuntime,omitempty"`
 }
 
-// KubeProxyImages is the data model for the kube-proxy image config
-// The kube-proxy image config is a JSON file
-// for example, vhdbuilder/packer/kube-proxy-images.json
+// KubeProxyImages is the data model for the kube-proxy image config.
+// The kube-proxy image config is a JSON file.
+// for example, vhdbuilder/packer/kube-proxy-images.json.
```
Comment (cameronmeissner) [inline pkg/vhdbuilder/datamodel/component_configs.go:35]: broken sentence

File: pkg/vhdbuilder/datamodel/component_configs.go [hunk 4]
```diff
@@ -82,12 +82,14 @@ func NewComponentsFromFile(path string) (*Components, error) {
...
-// from the given Components object
-// The image names are generated from the given Components object
-// and the given downloadURL
-// The image names are generated by replacing the * in the downloadURL
-// with the image tag
+/*
+ToImageList returns a list of image names
+from the given Components object.
+The image names are generated from the given Components object
+and the given downloadURL.
```
Comment (cameronmeissner) [inline pkg/vhdbuilder/datamodel/component_configs.go:89]: broken sentence

File: pkg/vhdbuilder/datamodel/component_configs.go [hunk 5]
```diff
@@ -111,10 +113,12 @@ func (c *Components) ToImageList() []string {
 
 // begins KubeProxyImages
 
-// NewKubeProxyImagesFromFile loads kube-proxy image config from the given file
-// and returns a KubeProxyImages object
-// The given file should be a KubeProxyImages object
-// The given file should be in JSON format
+/*
+NewKubeProxyImagesFromFile loads kube-proxy image config from the given file
+and returns a KubeProxyImages object.
```
Comment (cameronmeissner) [inline pkg/vhdbuilder/datamodel/component_configs.go:118]: broken sentence

File: pkg/vhdbuilder/datamodel/component_configs.go [hunk 6]
```diff
@@ -111,10 +113,12 @@ func (c *Components) ToImageList() []string {
...
+The given file should be a KubeProxyImages object.
```
Comment (cameronmeissner) [inline pkg/vhdbuilder/datamodel/component_configs.go:119]:
```suggestion
The given file should be a KubeProxyImages object, and should be in JSON format.
```

File: pkg/vhdbuilder/datamodel/component_configs.go [hunk 7]
```diff
@@ -126,12 +130,14 @@ func NewKubeProxyImagesFromFile(path string) (*KubeProxyImages, error) {
...
 
-// ToImageList returns a list of image names
-// from the given KubeProxyImages object
-// The image names are generated from the given KubeProxyImages object
-// and the given downloadURL
-// The image names are generated by replacing the * in the downloadURL
-// with the image tag
+/*
+ToImageList returns a list of image names
+from the given KubeProxyImages object.
```
Comment (cameronmeissner) [inline pkg/vhdbuilder/datamodel/component_configs.go:135]: broken sentence

**Review by ganeshkumar5699 — APPROVED** [review]

**Review by cameronmeissner — APPROVED** [review]

**Pull request merged**